10-3/sass/style.scss

```diff
@@ -19,9 +19,6 @@
 				width: 600px;
 				height: 400px;
 				float: left;
-				img {
-					width: 600px;
-				}
```

Comment:
te szerokosc tutaj mozesz przywrocic, bo jednemu z obrazkow po zmniejszeniu wychodzi mniej niz 600px. Oczywiscie to jest rozwiazanie robocze, bo normalnie mialabyc obrazki odpowiedniej wielkosci